gob/k8s/gen-growthhack/rc.yml

```diff
@@ -23,9 +23,10 @@ spec:
 
       containers:
         - name: svc
-          image: gobsvc/gen:growthhack
-          args:
-            - "--srv=:8080"
+          image: gcr.io/buoyant-hosted/grpc-gob:0.0.1-growthhack
```

Comment:
sgtm, but I'm not a member of the `gobsvc` organization cc/ @olix0r 
or we can put it under buoyant like the other examples do, either works for me